cmd/capabilities/DEPLOY-WORKERS.sh

```diff
@@ -0,0 +1,71 @@
+set -e
+
+gcloud config set project ${REGISTRY_PROJECT_IDENTIFIER}
+
+# Get credentials for kubectl
+gcloud container clusters get-credentials registry-backend --zone us-central1-a
+
+registry_ingress_ip=$(kubectl get service registry-backend -o jsonpath="{.status.loadBalancer.ingress[0].ip}")
+registry_service_port=$(kubectl get service registry-backend -o jsonpath="{.spec.ports[0].port}")
+if [ -z "${registry_ingress_ip}" ]; then
+  echo "Please make sure the Registry service is deployed to GKE and exposed externally."; exit 1
+fi
+
+# TODO: Use kubernets DNS
+export APG_REGISTRY_ADDRESS="${registry_ingress_ip}:${registry_service_port}"
+
+project_number=$(gcloud projects describe ${REGISTRY_PROJECT_IDENTIFIER} --format="value(projectNumber)")
+
+# Set correct IAM permissions to default SAs
+gcloud projects add-iam-policy-binding ${REGISTRY_PROJECT_IDENTIFIER} \
+       --member="serviceAccount:${project_number}-compute@developer.gserviceaccount.com" \
+       --role=roles/cloudtasks.enqueuer
+
+gcloud projects add-iam-policy-binding ${REGISTRY_PROJECT_IDENTIFIER} \
+       --member="serviceAccount:${project_number}-compute@developer.gserviceaccount.com" \
+       --role=roles/pubsub.subscriber
+
+# Registry worker
+deployment=$( kubectl get deployment registry-worker || true)
```

Comment:
Maybe you don't need "|| true" here? With $() it runs the command in a subprocess, so it won't exit the current process without || true. I might misunderstood the purpose of "|| true" here

Reply:
Yes, I am using `|| true` to prevent the script from exiting. Tried the suggestion, the script exits if `|| true` is absent.